cmake/find_llvm.cmake

```diff
@@ -9,7 +9,7 @@ if (ENABLE_EMBEDDED_COMPILER)
 
     if (LLVM_FOUND)
         # Remove dynamically-linked zlib and libedit from LLVM's dependencies:
-        set_target_properties(LLVMSupport PROPERTIES INTERFACE_LINK_LIBRARIES "-lpthread;LLVMDemangle;stdc++")
+        set_target_properties(LLVMSupport PROPERTIES INTERFACE_LINK_LIBRARIES "-lpthread;LLVMDemangle")
```

Comment:
It doesn't removed for whatever reason, even more: we have many new dependencies (all of them must be linked statically).

```
$ ldd ./clickhouse
        linux-vdso.so.1 =>  (0x00007ffd55383000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f1a7ce24000)
        libLTO.so.7 => /usr/local/lib/libLTO.so.7 (0x00007f1a79e1f000)
        librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f1a79c17000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f1a79a13000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f1a7970a000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f1a79341000)
        /lib64/ld-linux-x86-64.so.2 (0x000055ff1da45000)
        libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f1a79127000)
        libtinfo.so.5 => /lib/x86_64-linux-gnu/libtinfo.so.5 (0x00007f1a78efe000)
        libc++.so.1 => /usr/local/lib/../lib/libc++.so.1 (0x00007f1a78c33000)
        libc++abi.so.1 => /usr/local/lib/../lib/libc++abi.so.1 (0x00007f1a789fd000)
        libgcc_s.so.1 => /usr/local/lib64/libgcc_s.so.1 (0x00007f1a787e5000)
```

Reply:
Can't reproduce this, which version of LLVM are you using? Perhaps something is wrong with Compiler-X.Y.Z/CMakeLists.txt?